cmd/podman/errors_remote.go

```diff
@@ -0,0 +1,43 @@
+// +build remoteclient
+
+package main
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"syscall"
+
+	"github.com/containers/libpod/cmd/podman/varlink"
+	"github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+)
+
+func outputError(err error) {
```

Comment:
Not in this PR, but it might be possible to avoid this by having a ErrorReason interface (that provides a `Reason() string` method), having these errors implement that method, and trying to cast to an ErrorReason. Might be more maintanable?